openBCIBoard.js

```diff
@@ -220,8 +264,17 @@ function OpenBCIFactory() {
      * @author AJ Keller (@pushtheworldllc)
      */
     OpenBCIBoard.prototype.streamStop = function() {
-        this.streaming = false;
-        return this.write(k.OBCIStreamStop);
+        return new Promise((resolve,reject) => {
+            if(!this.streaming) reject('Error [.streamStop()]: No stream to stop');
+            this.streaming = false;
+            this.write(k.OBCIStreamStop)
+                .then(() => {
+                    setTimeout(() => {
+                        resolve();
+                    }, 50); // allow time for command to get sent
```

Comment:
Made it 10ms, now since the device is polling the host every 4 ms with new data, we should be fine here.